server/index.js

```diff
@@ -73,6 +74,52 @@ app.get('/api/get/waitlist', (req, res, next) => {
     })
     .catch(err => next(err));
 });
+
+app.get('/api/get/tables', (req, res, next) => {
+  const sql = `select * from "tables"
+              order by "tableNumber" asc`;
+  db.query(sql)
+    .then(result => {
+      const tableListResult = result.rows;
+      res.json(tableListResult);
+    });
+});
+
+app.put('/api/put/table/:id', (req, res, next) => {
```

Comment:
you can remove `get`, `put`, `delete`, etc from your route paths, the method is already matched for you by Express.js

```js
app.put('/api/table/:id', (req, res, next) => {
```